src/components/DashBoard.tsx

```diff
@@ -235,7 +241,12 @@ const Dashboard: React.FC = () => {
       </Drawer>
       <main className={classes.content}>
         <Switch>
-          <Route path="/ProposalEdit/:proposalPk" component={ProposalEdit} />
+          <Page
```

Comment:
I can see there's only one route which is now a `Page`. Is that right?

Reply:
Yeah, I noticed this and have closed this PR due to another issue anyway so will rectify this as well. All routes will be Pages.